catalog/spatial/registry/registry-rest-endpoint/pom.xml

```diff
@@ -50,11 +50,7 @@
         <dependency>
             <groupId>org.apache.commons</groupId>
             <artifactId>commons-lang3</artifactId>
-            <version>3.4</version>
-        </dependency>
-        <dependency>
-            <groupId>org.codice.ddf.registry</groupId>
```

Comment:
We still use RegistryPackageType in the endpoint so i think we still need the schema bindings